### DISCUSSION TITLE
Notes on running pre-built Radioss

### DISCUSSION THREAD

**Discussion by yw5aj** [OpenRadioss/OpenRadioss#412] [Contributors]
Hi all,

Just a quick note - it seems like if we're running the pre-built Radioss from [here](https://github.com/OpenRadioss/OpenRadioss/releases), then instead of `export RAD_CFG_PATH=$OPENRADIOSS_PATH/hm_cfg_files` it should be `export RAD_CFG_PATH=$OPENRADIOSS_PATH/cfg`. In addition, I needed to install `libapr` (I did `sudo apt-get install libaprutil1` for Ubuntu) before running.

Hope it helps!

Shawn

**Comment by PaulAltair** [comment]
Thanks for pointing this out, it is mentioned in INSTALL.md but easy to miss

**Reply by yw5aj** [reply]
Hi Paul, thank you for your reply! Maybe I missed something - if you are referring to [this document](https://github.com/OpenRadioss/OpenRadioss/blob/main/INSTALL.md), I don't think that was mentioned though. For example, for the `cfg` the only mention was `hm_cfg_files`; and I did not find anything on `apr` on that page.
